conf/nomad/presto.hcl

```diff
@@ -132,6 +132,9 @@ job "${nomad_job_name}" {
 
     task "server" {
       driver = "docker"
+      vault {
+              policies = ["kv-secret"]
+            }
```

Comment:
Does this supposed to be part of the nomad job? Could you achieve the same via terraform-vault-provider https://registry.terraform.io/providers/hashicorp/vault/latest/docs/resources/policy

Reply:
Well, Fredrik mentioned in the issue description #36  to make use of **vault stanza** to retrieve secrets so i implemented it likewise : https://www.nomadproject.io/docs/job-specification/vault

Reply:
updates. Implementation is correct to be able to avoid storing secrets in terraform state.